### PR TITLE
Update partition layouts for 22-xinyi-pc-classic

### PR DESCRIPTION
**Pull request by medicalwei**
Update the partition sizes according to the info from our customer.

### CODE REVIEW

File: gadget.yaml [hunk 1]
```diff
@@ -19,16 +19,21 @@ volumes:
...
       - # Recovery partition
         type: E3C9E316-0B5C-4DB8-817D-F92DF00215AE
-        size: 8192000000
-        offset: 250000384
+        size: 12884901888
+      - name: ubuntu-seed
+        role: system-seed
+        filesystem: ext4
+        type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        size: 1200M
```
Comment (xnox) [inline gadget.yaml]: having partition ubuntu-seed as ext4 is unusual. And having it as not ESP is unusual. I am not sure if it is valid to have ubuntu-seed with a role system-seed, whilst ESP is marked as system-seed-null.

Has this been checked with snapd team? @alfonsosanchezbeato

**Review by xnox — COMMENTED** [review]
I would like to request ubuntu-core/snapd team to review this change.

**Comment by xnox** [comment]
@alfonsosanchezbeato @pedronis @mvo5 do things proposed here adhere to gadget specifications?

**Comment by mwhudson** [comment]
I think it would be best to get the new sizes landed asap and then have a proper conversation about ubuntu-seed in a separate PR.

**Comment by xnox** [comment]
> I think it would be best to get the new sizes landed asap and then have a proper conversation about ubuntu-seed in a separate PR.

can i please have a new PR, which just updates the sizes of existing things and/or gaps&offsets without introducing `role: system-seed` (the gap is there currently, right?)

**Comment by medicalwei** [comment]
I updated the PR so that only offsets and sizes are changed.  The gap is now 1200MiB between recovery and ubuntu-boot

**Review by xnox — APPROVED** [review]

**Pull request merged**